support-frontend/assets/pages/new-contributions-landing/components/ContributionTypeTabs.jsx

```diff
@@ -56,7 +56,7 @@ const mapDispatchToProps = (dispatch: Function) => ({
 function ContributionTypeTabs(props: PropTypes) {
   const contributionTypes = props.contributionTypes[props.countryGroupId];
```

Comment:
If we want to avoid making this part of Redux, we could check to see if the campaign contribution types exist here. This would be consistent with the other campaign configuration checks, which are all done inside the component using them. I don't know if that's the best idea, but it's working to keep the campaign configuration outside the state

Reply:
the `contributionTypes` settings are also needed during initialisation in `contributionsLandingInit.js`, and it would be better to only calculate them once

Reply:
ah right, i forgot they're needed earlier than at the point of rendering